src/formats/chips.hpp

```diff
@@ -142,7 +142,7 @@ class CHIPSProcessor : virtual public PacketProcessor {
 	    */
 	    //if( pkt->src < 8 ) { // HACK TESTING
 	    //for( ; chan<32; ++chan ) { // HACK TESTING
-	    for( ; chan<pkt->nchan; ++chan ) { // HACK TESTING
+	    for( ; chan<32; ++chan ) { // HACK TESTING pkt->nchan
```

Comment:
Clean up. Why are we only unpacking 32 channels? Should this be a parameter passed in, instead of hard coded? On slack @HaraKrish said he was unpacking 132 channels, but this looks like it's only 32.

Reply:
I'm unpacking 90 channels, this one is out-of-date.

Reply:
I still find it strange that we have to limit how many channels are unpacked to get things running.